package.json

```diff
@@ -16,9 +16,6 @@
   },
   "homepage": "https://github.com/oppia/oppia#readme",
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.1102.10",
```

Comment:
@kevintab95 Why are these changes part of this PR?

Reply:
Sorry, these are unexpected, should be fixed now.